crates/bevy_render/src/lib.rs

```diff
@@ -10,9 +11,10 @@ pub mod renderer;
 pub mod shader;
 pub mod texture;
 
-mod entity;
 pub use once_cell;
 
+pub use crate::base::Msaa;
```

Comment:
I think we can/should drop this line as msaa is already publicly exported via `crate::render_graph::base::Msaa`. I honestly forgot `base` wasn't a top level module when i wrote my last comment.

Reply:
Oh I didn't realize it was already exported; I'll get rid of that.